.gitignore

```diff
@@ -67,6 +67,21 @@ target/
 # VS Code
 .vscode/
 
+# OSX files
+.DS_Store
```

Comment:
I'm a bit on the fence about stuff like this (and editor specific files). On one hand it keeps people from accidentally add binary blobs to the repo, on the other hand it's tied to ones setup.

I use a global gitignore, `~/.gitignore` I believe, for stuff like this. Maybe we should add something to the README instead?

Reply:
NEj, jag tycker den ska vara med

Reply:
Eller vill du committa in dina DS_Store-filer / att de skall synkas till någon annans dator?